Author: Denis Kristak (Inuits)
Change build backend to setuptools.build_meta to avoid using hatchling (not supported on this toolchain).
diff -ruN plumbum-1.8.2_orig/pyproject.toml plumbum-1.8.2/pyproject.toml
--- plumbum-1.8.2_orig/pyproject.toml	2023-05-30 11:30:45.000000000 +0100
+++ plumbum-1.8.2/pyproject.toml	2023-10-17 01:12:34.635282386 +0100
@@ -1,9 +1,6 @@
 [build-system]
-requires = [
-    "hatchling",
-    "hatch-vcs",
-]
-build-backend = "hatchling.build"
+requires = []
+build-backend = "setuptools.build_meta"
 
 
 [project]
@@ -13,7 +10,7 @@
 authors = [{ name="Tomer Filiba", email="tomerfiliba@gmail.com" }]
 license = { file="LICENSE" }
 requires-python = ">=3.6"
-dynamic = ["version"]
+version = "1.8.2"
 dependencies = [
     "pywin32; platform_system=='Windows' and platform_python_implementation!='PyPy'",
 ]
